src/components/Articles.js

```diff
@@ -0,0 +1,13 @@
+import  React from "react";
+import Sections from "./article_table/table_body";
+function articles() {
```

Comment:
Might be the best place to add the data in the form of an object and pass on the information to section component

Reply:
Passed data in the form of object. Please check and review it.

![image](https://user-images.githubusercontent.com/89765443/186846934-ad754474-6d89-4060-adbe-6ca60331f3b6.png)